fdbserver/Knobs.cpp

```diff
@@ -428,7 +428,9 @@ ServerKnobs::ServerKnobs(bool randomize, ClientKnobs* clientKnobs, bool isSimula
 	init( MAX_TRANSACTIONS_PER_BYTE,                            1000 );
 
 	init( MIN_FREE_SPACE,                                        1e8 );
-	init( MIN_FREE_SPACE_RATIO,                                 0.05 );
+	init( START_MIN_FREE_SPACE_RATIO,                           0.26 );
```

Comment:
How is the value chosen for `START_MIN_FREE_SPACE_RATIO ` and `END_MIN_FREE_SPACE_RATIO `?
The rationale can be useful for DBA to customize the param.

Reply:
The end is the same as the old limit, the begin is just before the point we start to worry about free space (.20)